configuration_test.go

```diff
@@ -0,0 +1,268 @@
+package govh
+
+// precedence des fichiers de configuration
+// chargement de la configuration depuis les variables d'env
+// chargement manauel de la configuration
+// erreur s'il manque des paramètres
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+//
+// Utils
+//
+
+var home string
+
+func setup() {
+	systemConfigPath = "./ovh.unittest.global.conf"
+	userConfigPath = "/.ovh.unittest.user.conf"
+	localConfigPath = "./ovh.unittest.local.conf"
+	home, _ = currentUserHome()
+}
+
+func teardown() {
+	os.Remove(systemConfigPath)
+	os.Remove(home + userConfigPath)
+	os.Remove(localConfigPath)
+}
+
+//
+// Tests
+//
+
+func TestConfigFromFiles(t *testing.T) {
+	// Write each parameter to one different configuration file
+	// This is a simple way to test precedence
+
+	// Prepare
+	ioutil.WriteFile(systemConfigPath, []byte(`
```

Comment:
I prefer avoiding as much as possible external dependencies. The more we have, the harder it is to guarantee the code quality. Having a couple of files potentially left behind on a trashable build host seems an acceptable trace-off to me.